data-prepper-plugins/geoip-prepper/src/main/java/com/amazon/dataprepper/plugins/prepper/geoip/provider/LocationData.java

```diff
@@ -16,8 +16,8 @@
     private final String REGION_CODE_FIELD = "geo.region_code";
     private final String TIMEZONE_FIELD = "geo.timezone";
     private final String LOCATION_FIELD = "geo.location";
-    private final String LATITUDE_FIELD = "geo.location.lat";
-    private final String LONGITUDE_FIELD = "geo.location.lon";
+    private final String LATITUDE_FIELD = "geo.latitude";
+    private final String LONGITUDE_FIELD = "geo.longitude";
```

Comment:
any reason for this change ?

Reply:
fields named geo.location.lat and geo.location.lon were causing issues with indexing the geo.location geo_point in Elasticsearch

Reply:
Was the data not persisting into index ? What was the issue ?